modules/core/shared/src/test/scala/gem/config/Arbitraries.scala

```diff
@@ -122,6 +121,15 @@ trait Arbitraries {
       } yield StaticConfig.GmosSouth(c, s)
     )
 
+  implicit val arbGnirsWellDepth: Arbitrary[GnirsWellDepth] =
+    Arbitrary(
+      Gen.oneOf(GnirsWellDepth.Shallow,
+                GnirsWellDepth.Deep)
+    )
```

Comment:
There is an `Enumerated` instance for `GnirsWellDepth` so I don't think this needs to be explicitly defined if `ArbEnumerated.arbEnumerated` is in scope.